README.md

```diff
@@ -41,25 +42,110 @@ SCENARIOSURL | Url to the scenario JSON file to randomize
 LOCAL | `true` for local (default false)
 
 
-#### On windows
+### On windows
+
 ```sh
 set SEARCHTOKEN=value
 set UATOKEN=value
 set SCENARIOSURL=value
 set LOCAL=true #if scenariosurl is a local path
 ```
-#### On MAC
+
+### On MAC
+
 ```sh
 export SEARCHTOKEN = value
 export UATOKEN = value
 export SCENARIOSURL = value
 export LOCAL=true #if scenariosurl is a local path
 ```
 
+## Configuration Files
+
+### Documentation
+
+- [How to build scenarios](http://coveo.github.io/uabot/scenario.html)
+- [Examples of scenarios](https://github.com/coveo/uabot/tree/master/scenarios_examples)
+
+### Visual Studio Code Snippets
```

Comment:
Please move this section at the very bottom. I feel the installation details are more important to have on top.

Below the How to use but above the developer details.